### DISCUSSION TITLE
[mini.surround]: gza invalid in normal mode

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3647] [Q&A]
While using the `mini-surround` plugin, I found that gza (formerly gsa) doesn't run in normal mode, it works fine in visual mode.

**Comment by folke** [comment]
It's gsa, unless you enabled leap

**Reply by iamxiaojianzheng** [reply]
yes, i enable leap. so gsa became gza.

**Reply by dpetka2001** [reply]
It works for me. Are you pressing the corresponding textobject-motion after `gza`? For example to surround around a word you would have to press `gzaiw` to surround the word you want. The plugin expects a textobject-motion after the initial keymap. See `:h text-objects` for more info.

**Reply by iamxiaojianzheng** [reply]
Thank you for your reply. 

From your answer, I know why my operation failed.
